### ISSUE TITLE
Two deprecated npm packages...

### ISSUE THREAD

**Issue by richb-hanover** [PRQL/prql#2817]
### What's up?

Not quite a bug report... The npm install process displays two warnings about deprecated pages. Here's the message:

```
vscode ➜ /workspaces/prql/web/playground (main) $ npm i
npm WARN deprecated stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility
npm WARN deprecated svgo@1.3.2: This SVGO version is no longer supported. Upgrade to v2.x.x.
```

Both these packages seem to be dependencies of `wasm-react-scripts@5.0.3` in _web/playground/package.json_

[npmjs.com](http://npmjs.com/) shows a link to this page https://www.npmjs.com/package/wasm-react-scripts  and a link to a this Github repo: https://github.com/Cosmian/create-react-app 

But I can't figure out where the "wasm-react-scripts" reside to send an informative note to the maintainer. Does anyone have insights? Thanks.

**Comment by max-sixty** [comment]
I had a quick look but I agree I couldn't find the package, possibly the Readme is from a repo which this forked from

I do see a note in our repo re https://github.com/PRQL/prql/blob/3a50d91d764440a98c73605a702259c5331ac2a3/web/playground/README.md#L24-L25, linking to a PR which never got merged, so we're on a fork of the official repo. I don't have much context here; very open to more canonical ways of doing this.

**Comment by aljazerzen** [comment]
Yup, when I built the playground, I needed to switch to that fork to get our wasm and duckdb to work.

The deprecated dependencies are not worth updating:
- `stable` is just saying "this package is not really needed anymore",
- `svgo` is saying "we won't update this package anymore",

I also got a few more warnings:
- `rollup-plugin-terser`,
- `sourcemap-codec`,
- `workbox-cacheable-response`,

... all saying "this package has been moved".

So I don't think this is a concern.

The proper solution would be to rewrite the playground, or at least replace the create-react-app framework.
